pkg/resources/stream_acceptance_test.go

```diff
@@ -71,3 +71,67 @@ resource "snowflake_stream" "test_stream" {
 `
 	return fmt.Sprintf(s, name, name, name)
 }
+
+func TestAcc_Stream_AppendOnly(t *testing.T) {
```

Comment:
Let's move this up to where the other test is `TestAcc_Stream`. Around line 33

Reply:
Just like we do here: https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/main/pkg/resources/schema_grant_acceptance_test.go#L30-L35

Reply:
I've moved the test into the existing method and updated the `streamConfig` function to allow the caller to specify whether the stream is append only. I've tried running the acceptance tests both locally and through Travis, but I seem to be missing something in the configuration. Are there docs for how to run the acceptance tests so I can verify they work?

Reply:
Never mind, I managed to execute them using Travis CI. I wasn't setting the Go version correctly and it was defaulting to 1.10. The `TestAcc_Stream` test passed.